src/main/java/no/entur/uttu/osrm/OsrmService.java

```diff
@@ -0,0 +1,135 @@
+package no.entur.uttu.osrm;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.mizosoft.methanol.Methanol;
+import com.github.mizosoft.methanol.MutableRequest;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.net.http.HttpClient;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OsrmService {
+
+  private static final Logger logger = LoggerFactory.getLogger(OsrmService.class);
+
+  @Value("${uttu.osrm-api}")
```

Comment:
I'd change this key to `uttu.routing.osrm.endpoint` instead, mainly because it reflects the generic name of the feature better.